Connectors/CSi/Speckle.Connectors.ETABSShared/HostApp/ETABSSendCollectionManager.cs

```diff
@@ -0,0 +1,117 @@
+using Speckle.Connectors.CSiShared.HostApp;
+using Speckle.Converters.Common;
+using Speckle.Converters.CSiShared;
+using Speckle.Sdk.Models;
+using Speckle.Sdk.Models.Collections;
+
+namespace Speckle.Connectors.ETABSShared.HostApp;
+
+/// <summary>
+/// ETABS-specific collection manager that organizes structural elements by story and type.
+/// </summary>
+/// <remarks>
+/// Creates a two-level hierarchy:
+/// 1. Story level (e.g., "Story 1", "Story 2")
+/// 2. Element type level with ETABS-specific categorization:
+///    - Frames: Columns, Beams, Braces
+///    - Shells: Walls, Floors, Ramps
+///
+/// Elements without story assignment are placed in "Unassigned" collection.
+/// Uses caching to maintain collection references and prevent duplicates.
+/// </remarks>
+public class ETABSSendCollectionManager : CSiSendCollectionManager
+{
+  public ETABSSendCollectionManager(IConverterSettingsStore<CSiConversionSettings> converterSettings)
+    : base(converterSettings) { }
+
+  // TODO: This is gross. Too many strings. Improve as part of next milestone. Out of scope of "First Send".
+  public override Collection AddObjectCollectionToRoot(Base convertedObject, Collection rootObject)
+  {
+    var properties = convertedObject["properties"] as Dictionary<string, object>;
```

Comment:
hmm, why we need properties in collection manager? this looks like anti-pattern with other connectors. happy to chat and understand if i am missing something on ETABS side

Reply:
Definitely room for improvement here. Happy to sync